admin_guide/managing_projects.adoc

```diff
@@ -94,3 +102,60 @@ or:
 To request a new project, fill out the project request form located at
 https://internal.example.com/openshift-project-request.
 ----
+
+[[using-node-selectors]]
+== Using Node Selectors
+
+Node selectors are used in conjunction with labels to control pod placement. 
```

Comment:
@sosiouxme Yes, that's why i was asking if these labels are the same as the other labels. I think if we refer to them as "node labels" it might lead to confusion within people who have the same or less understanding as me... There's the note box after this that talks more about the labels. I think that's enough. Let me know if you disagree.